src/main/java/com/project/momskitchen/backend/persistence/impl/MomsKitchenDB.java

```
@@ -0,0 +1,53 @@
+package com.project.momskitchen.backend.persistence.impl;
+
+import java.sql.*;
+import com.project.momskitchen.backend.exceptions.MomsPersistenceException;
+import com.project.momskitchen.backend.model.User;
+
+public class MomsKitchenDB {
+    //Direccion del DB de prueba
+    static String urlDatabase = "";
+    static String usuarioDb = "";
+    static String passwordDb= "";
+    static Connection c = null;
+
+    public void realizaConexion() throws MomsPersistenceException{
+
+        try {
+            Class.forName("org.postgresql.Driver");
+            c = DriverManager.getConnection(urlDatabase,  usuarioDb, passwordDb);
+            System.out.println("La conexion se realizo sin problemas!");
+        } catch (ClassNotFoundException e) {
+        } catch (SQLException e) {
```

Never leave empty catch blocks. It is even preferred to propagate the original exception